### PR TITLE
Kyma installation on a cluster

### PR DESCRIPTION
**Pull request by crabtree**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**Description**

This PR introduces description of the installation steps for kyma on a cluster.

### CODE REVIEW

File: installation/README.md [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+
+### 2. The `installer-config.yaml.tpl` template
+
+The Kyma installation process requires installation data specified in the `installer-config.yaml` file. Copy the `installer-config.yaml.tpl` template, rename it to `installer-config.yaml`, and fill in the placeholders.
+
+- `__TLS_CERT__` Placeholder for TLS certificate
+- `__TLS_KEY__` Placeholder for TLS certificate key
+- `__REMOTE_ENV_CA__` Placeholder for Remote Environments CA
+- `__REMOTE_ENV_CA_KEY__` Placeholder for Remote Environments CA key
+- `__EXTERNAL_IP_ADDRESS__` Placeholder for IP address for Kyma Ingress
```
Comment (pbochynski) [inline installation/README.md]: The IP address should be used only when provided. It should not be required.

File: installation/README.md [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+
+Kubernetes API Server is configured in the following manner:
+
+```
+"apiServerConfig": {
+    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota,PodPreset",
+    "--runtime-config": "batch/v2alpha1=true,settings.k8s.io/v1alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
+    "--cors-allowed-origins": ".*",
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--oidc-issuer-url": "https://dex.__DOMAIN_NAME__",
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/README.md]: Explain that __DOMAIN_NAME__ is a placeholder thus this entire thing is a template.

File: installation/README.md [hunk 3]
```diff
@@ -1,3 +1,95 @@
 # Kyma Installer
 
-The Kyma installer is a proprietary solution based on the Kubernetes operator. See the [Local Kyma installation](../docs/kyma/docs/031-gs-local-installation.md) document for basic information on how to use the installer and run Kyma locally.  
+The Kyma installer is a proprietary solution based on the Kubernetes operator. See the [Local Kyma installation](../docs/kyma/docs/031-gs-local-installation.md) document for basic information on how to use the installer and run Kyma locally.
```
Comment (kazydek) [inline installation/README.md]: Can we add an `## Overview` section above the text?

File: installation/README.md [hunk 4]
```diff
@@ -1,3 +1,95 @@
...
+- `__ENABLE_ETCD_BACKUP_OPERATOR__` Placeholder for enabling/disabling `etcd` backup operator ("true", "false")
+- `__ECTD_BACKUP_ABS_CONTAINER_NAME__` Placeholder for Azure Blob Storage name for `etcd` backups, may be left blank when backup operator is disabled
+
+Once all the required placeholders are filled in, run the following command to provide the cluster with the installation data:
+
+```
+kubectl apply -f installer-config.yaml
+```
+
+### 3. Deploy the `Installer` component
```
Comment (kazydek) [inline installation/README.md]: ``3. Deploy the `Installer` component.``

File: installation/README.md [hunk 5]
```diff
@@ -1,3 +1,95 @@
...
+
+To deploy the `Installer` component on your cluster, run the following command:
+
+```
+kubectl apply -f installation/resources/installer.yaml -n kyma-installer
+```
+
+### 4. Trigger the installation
```
Comment (kazydek) [inline installation/README.md]: `4. Trigger the installation.`

File: installation/README.md [hunk 6]
```diff
@@ -1,3 +1,95 @@
...
+
+To trigger the installation of Kyma, you will need a Custom Resource file. Copy the `installer-cr.yaml.tpl` file, rename it to `installer-cr.yaml`, and fill in the placeholders:
```
Comment (kazydek) [inline installation/README.md]: you will need -> `you need `
fill in the placeholders: -> `fill in these placeholder values:`

File: installation/README.md [hunk 7]
```diff
@@ -1,3 +1,95 @@
...
+
+- `__VERSION__` Version number of kyma to be installed
+- `__URL__` URL to the kyma package to be installed
+
+Once the file is ready, run the following command to trigger the installation.
```
Comment (kazydek) [inline installation/README.md]: installation. -> `installation:`

File: installation/README.md [hunk 8]
```diff
@@ -1,3 +1,95 @@
...
+
+```
+kubectl apply -f installer-cr.yaml
+```
+
+To check the progress of the installation process, inspect the Custom Resource:
```
Comment (kazydek) [inline installation/README.md]: inspect -> `verify`

File: installation/README.md [hunk 9]
```diff
@@ -1,3 +1,95 @@
...
+kubectl apply -f installation/resources/installer.yaml -n kyma-installer
+```
+
+### 4. Trigger the installation
+
+To trigger the installation of Kyma, you will need a Custom Resource file. Copy the `installer-cr.yaml.tpl` file, rename it to `installer-cr.yaml`, and fill in the placeholders:
+
+- `__VERSION__` Version number of kyma to be installed
+- `__URL__` URL to the kyma package to be installed
+
```
Comment (kazydek) [inline installation/README.md]: ``
- `__VERSION__`  for the version number of Kyma to install
- `__URL__` for the URL to the Kyma package to install
```

**Pull request merged**